cmd/transfer-pvc/transfer-pvc.go

```diff
@@ -413,11 +431,32 @@ func (t *TransferPVCCommand) run() error {
 		log.Fatal(err, "failed to find node name")
 	}
 
+	clientPodSecCtx, err := getRsyncClientPodSecurityContext(srcClient, srcPVC.Namespace)
+	if err != nil {
+		log.Fatal(err, "error creating security context for rsync server")
+	}
+
 	_, err = rsynctransfer.NewClient(
 		context.TODO(),
 		srcClient, srcPVCList, stunnelClient, e, logger, "rsync-client", labels, nil, rsyncPassword,
 		transfer.PodOptions{
 			NodeName: nodeName,
+			CommandOptions: rsynctransfer.NewDefaultOptionsFrom(
+				verify(t.Verify),
+				restrictedContainers(true),
+				verbose(true),
+			),
+			ContainerSecurityContext: corev1.SecurityContext{
+				Privileged: &falseBool,
+				Capabilities: &corev1.Capabilities{
+					Drop: []corev1.Capability{"ALL"},
+				},
+				RunAsNonRoot: &trueBool,
+				RunAsUser:    clientPodSecCtx.RunAsUser,
+			},
```

Comment:
```suggestion
                                AllowPrivilegeEscalation: &falseBool,
				SeccompProfile: &corev1.SeccompProfile{
								Type: "RuntimeDefault",
							},
			},
```

I think we need these two field values on container/pod SC as well.

Reply:
@JaydipGabani Thanks! didn't know about these two

Reply:
Does this stuff exist in prior clusters? Do we need to be careful about what we apply to containers on the Source side for fear of them not coming up?

@jmontleon mentioned something to this effect recently and I just wonder if it applies here.

Reply:
I saw pods getting admitted in the 3.11 cluster with this `SecurityContext`, `SeccompProfile` field gets omitted as it doesn't exist.

Reply:
Cool. That's great news if we don't have to worry about mismatch between src/dest clusters for transfer-pvc.